src/screens/Home/index.js

```diff
@@ -9,21 +10,43 @@ import { VIEW_CONTENT_TYPE, MOCK_DATA } from './constants';
 
 function Home() {
   const [viewType, setViewType] = useState(VIEW_CONTENT_TYPE[0].id);
+  const [data] = useState(MOCK_DATA);
+  const [mapData, setMapData] = useState();
+
+  useEffect(() => {
+    const filteredData = MOCK_DATA.map((el) => ({
+      id: el.alpha2Code,
+      name: el.name,
+      nativeName: el.nativeName,
+      externalId: el._id,
+      emoji: el.flag.emoji,
+    }));
+    setMapData(filteredData);
+  }, []);
 
   return (
     <section className={styles.container}>
       <Header value={viewType} onChange={setViewType} />
-      {viewType === VIEW_CONTENT_TYPE[0].id ? (
-        <>
-          <Searcher />
-          <ul className={styles.content}>
-            {MOCK_DATA.map((el) => (
-              <Card key={el._id} data={el} />
-            ))}
-          </ul>
-        </>
-      ) : (
-        <Map />
+      <div
+        className={clsx({
+          [styles.hide]: viewType === VIEW_CONTENT_TYPE[1].id,
+        })}
+      >
+        <Searcher />
+        <ul className={styles.cardContent}>
+          {data.map((el) => (
+            <Card key={el._id} data={el} />
+          ))}
+        </ul>
+      </div>
+      {mapData && (
+        <div
+          className={clsx({
+            [styles.hide]: viewType === VIEW_CONTENT_TYPE[0].id,
+          })}
+        >
+          <Map data={mapData} />
```

Comment:
If map has a root div (which I can't see in this PR) you may avoid the parent div here and pass a className to Map